scripts/ensure-prisma2-version.ts

```diff
@@ -1,26 +0,0 @@
-/**
```

Comment:
Not needed anymore?

Reply:
Because this script was previously there to ensure that dev deps and defined peer deps were in sync. That was mainly to prevent us from forgetting to update the peer deps when upgrading prisma.
Now that there's no longer peer deps at all, the check is no longer needed.